sql/src/main/java/io/crate/executor/transport/TransportExecutor.java

```diff
@@ -429,10 +433,12 @@ public void addPhase(@Nullable ExecutionPhase executionPhase) {
              * to avoid race conditions.
              */
             public void addCollectExecutionPhase(@Nullable ExecutionPhase executionPhase) {
-                addPhase(executionPhase, collectNodeOperations);
+                addPhase(executionPhase, collectNodeOperations, true);
             }
 
-            private void addPhase(@Nullable ExecutionPhase executionPhase, List<NodeOperation> nodeOperations) {
+            private void addPhase(@Nullable ExecutionPhase executionPhase,
+                                  List<NodeOperation> nodeOperations,
+                                  boolean connectWithPreviousPhase) {
```

Comment:
please change name of that flag, it's missleading. A colelctPhase usually doesn't have a previous phase